climt/_components/gfs/component.py

```diff
@@ -521,6 +468,12 @@ def __del__(self):
         print("Done!")
 
 
+def make_positive(array):
```

Comment:
This name doesn't actually describe what the function does (I would assume return abs(array) based on the name). I'd suggest refactoring to something like "set_negatives_to_zero" or "zero_negatives".